components/TransactionHistory.jsx

```diff
@@ -0,0 +1,53 @@
+let React = require('react');
+
+let ReactBootstrap = require('react-bootstrap');
+let Table = ReactBootstrap.Table;
+
+let TransactionItem = require('./TransactionItem');
+
+// container of the transaction items
+let TransactionHistory = React.createClass({
+
+  render() {
+    let transactions = this.props.transactions.map((t, i) => {
+      return (
+        <TransactionItem
+          key={this.props.transactions.length - i - 1}
```

Comment:
I put only put this `key` attribute to avoid a warning from React:

    Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of TransactionHistory. See https://fb.me/react-warning-keys for more information.